Amplify/Categories/DataStore/Model/Collection/List+LazyLoad.swift

```diff
@@ -32,20 +32,12 @@ extension List {
 
         // if the collection has no associated field, return the current elements
         guard let associatedId = self.associatedId,
-              let associatedField = self.associatedField else {
+              let associatedField = associatedField else {
```

Comment:
looks like u removed the use of `self` for associatedField, do the same for the first?`let associatedId = associatedId` as well?